### DISCUSSION TITLE
Label shadow issue

### DISCUSSION THREAD

**Discussion by mamayadi** [darioielardi/flutter_speed_dial#152] [Q&A] [answered]
I cannot modify the shadow of the label here is the exemple:
![image](https://user-images.githubusercontent.com/28813940/70431985-7b984680-1a7e-11ea-84f5-018ebc32351a.png)

**Comment by darioielardi** [comment]
Unfortunately it's not possible to customize the default label shadow, but you can always provide your own label widget completely custom. Just use the labelWidget argument of the SpeedDialChild!

Let me know if I can close!

**Comment by prateekmedia** [comment, marked as answer]
Check the new version, it has everything fixed, Just added it for SpeedDialChild
